setup.cfg

```diff
@@ -1,2 +1,36 @@
+[metadata]
+name = pathspec
+version = 0.5.9
+author = Caleb P. Burns
+author_email = cpburnz@gmail.com
+license = MPL 2.0
+description = Utility library for gitignore style pattern matching of file paths.
+url = https://github.com/cpburnz/python-path-specification
+long_description = file: README.rst, CHANGES.rst
+long_description_content_type = text/x-rst
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Utilities
+
+[options]
+packages = find:
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+setup_requires = setuptools>=44; wheel; setuptools_scm[toml]>=3.4.3
```

Comment:
*wheel* shouldn't be a hard setup requirement because I only use that when publishing.

Reply:
In fact pip builds wheels, if you have pip, you have wheel (though inlined into pip).